static/js/syntaxModesRules.ts

```diff
@@ -714,6 +732,18 @@ function rule_string() {
 
 
 
+function rule_blank() {
+  return [{
+    regex: /\_\?\_/,
+    token: 'invalid',
```

Comment:
I am not really sure I like the word invalid here, can we use blank?

Reply:
Unfortunately I didn't choose this word at random.
It corresponds to a style defined by monokai, a style that has the property of being flashy.

I can modify the file `./static/vendor/theme-monokai.js` to try to create a style